test/test_job.js

```diff
@@ -218,16 +218,6 @@ describe('Job', function(){
       });
     });
 
-    it('can renew a previously taken lock', function(){
-      return job.takeLock().then(function(lockTaken){
-        expect(lockTaken).to.be.truthy;
-      }).then(function(){
-        return job.renewLock().then(function(lockRenewed){
-          expect(lockRenewed).to.be.truthy;;
-        });
-      });
-    });
-
     it('can release a lock', function(){
```

Comment:
We could - but is the only value in making `extendLock` public for the unit tests? We wouldn't want it to be used publicly, since external code should be manipulating the job locks.